diff --git a/src/vs/platform/product/common/product.ts b/src/vs/platform/product/common/product.ts
index c222f11ecab..4dd0877fc88 100644
--- a/src/vs/platform/product/common/product.ts
+++ b/src/vs/platform/product/common/product.ts
@@ -74,12 +74,16 @@ else {
 	}
 }
 
-// product.extensionsGallery = {
-// 	serviceUrl: 'https://open-vsx.org/vscode/gallery',
-// 	controlUrl: '',
-// 	extensionUrlTemplate: '',
-// 	resourceUrlTemplate: 'https://open-vsx.org/vscode/asset/{publisher}/{name}/{version}/Microsoft.VisualStudio.Services.VSIXPackage',
-// 	nlsBaseUrl: '',
-// };
-
+// todo should fix this
+Object.assign(product, {
+	extensionsGallery: {
+		serviceUrl: 'https://open-vsx.org/vscode/gallery',
+		itemUrl: 'https://open-vsx.org/vscode/item',
+		controlUrl: 'https://open-vsx.org/vscode/control',
+		recommendationsUrl: 'https://open-vsx.org/vscode/recommendations',
+		resourceUrlTemplate: 'https://open-vsx.org/vscode/asset/{publisher}/{name}/{version}/Microsoft.VisualStudio.Code.Manifest',
+		assetUrlTemplate: 'https://open-vsx.org/vscode/asset/{publisher}/{name}/{version}',
+		fallbackAssetUrlTemplate: 'https://open-vsx.org/vscode/fallback/{publisher}/{name}/{version}'
+	}
+});
 export default product;
